src/pybind/mgr/zabbix/module.py

```diff
@@ -174,7 +174,10 @@ def get_data(self):
 
         osd_stats = self.get('osd_stats')
         for osd in osd_stats['osd_stats']:
-            osd_fill.append((float(osd['kb_used']) / float(osd['kb'])) * 100)
+            if osd['kb'] == 0:
```

Comment:
if an OSD's capacity is 0 shall we just ignore it as it's marked out?

Reply:
@tchaikov Yes, that seems like a good thing to do. That would otherwise take the average down incorrectly